setup.py

```diff
@@ -6,7 +6,9 @@
 import os
 import platform
 from subprocess import check_call, check_output
+
 from setuptools import Extension, find_packages, setup
+
```

Comment:
Could you please remove whitespace changes

Reply:
Done